documentation/general/contributing.md

```diff
@@ -1,10 +1,26 @@
-# Welcome
+# Get started with contributions
```

Comment:
I changed this title to make it easier to understand the navigation sidebar. The title "Welcome" was really ambiguous there, and this is a clearer call-to-action.